### PR TITLE
docker images: add page

### PR DESCRIPTION
**Pull request by Aracki**
I have added `docker images` missing command.

### CODE REVIEW

File: pages/common/docker-images.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- List all Docker images with intermediate included:
+
+`docker images -a`
+
+- List all Docker images numeric IDs:
+
+`docker images -q`
+
+- List all Docker images not used by any container:
```
Comment (sbrl) [inline pages/common/docker-images.md]: Is there any way of deleting these automatically with this command?

Comment (Aracki) [reply pages/common/docker-images.md]: No. For that one would use `docker rmi`.

Comment (sbrl) [reply pages/common/docker-images.md]: Right. That  would be a separate page then.

File: pages/common/docker-images.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# docker images
+
+> Manage Docker images.
+> More information: <https://docs.docker.com/engine/reference/commandline/images/>.
+
+- List all Docker images:
+
+`docker images`
+
+- List all Docker images with intermediate included:
```
Comment (agnivade) [inline pages/common/docker-images.md]: intermediates ? Maybe better is - List all Docker images including intermediates.

File: pages/common/docker-images.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`docker images -a`
+
+- List all Docker images numeric IDs:
```
Comment (agnivade) [inline pages/common/docker-images.md]: We should try to match the `-q` with the description. Let us change it to - "List the output in quiet mode (only numeric IDs)"

Comment (Aracki) [reply pages/common/docker-images.md]: good suggestion 👍

File: pages/common/docker-images.md (hunk 1 above)
Comment (Aracki) [reply pages/common/docker-images.md]: there are many subcommands missing in **_tldr_**, but slowly we are getting there :)

**Pull request merged**